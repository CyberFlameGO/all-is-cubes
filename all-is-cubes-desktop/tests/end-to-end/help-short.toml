bin.name = "all-is-cubes"
args = "-h"
