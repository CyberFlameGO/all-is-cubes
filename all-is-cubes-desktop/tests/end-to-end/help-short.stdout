All is Cubes [DESKTOPVERSION]
Kevin Reid <kpreid@switchb.org>
Recursive voxel game.

USAGE:
    all-is-cubes [OPTIONS] [FILE]

ARGS:
    <FILE>    Existing save/document file to load. If not specified, a template will be used
              instead

OPTIONS:
        --display-size <WÃ—H>     Window size or image size, if applicable to the selected --graphics
                                 mode [default: auto]
        --duration <SECONDS>     Length of time to simulate.
    -g, --graphics <mode>        Graphics/UI mode [default: window]
    -h, --help                   Print help information
        --no-config-files        Ignore all configuration files, using only defaults and command-
                                 line options
    -o, --output <FILE>          Output PNG file name for 'record' mode
        --precompute-light       Fully calculate light before starting the game
    -t, --template <TEMPLATE>    Which world template to use [default: demo-city] [possible values:
                                 menu, blank, fail, demo-city, dungeon, atrium, cornell-box,
                                 physics-lab, menger-sponge, lighting-bench[..]]
    -v, --verbose                Additional logging to stderr
    -V, --version                Print version information
